test/src/api/commands/web-element/testIsPresent.js

```diff
@@ -0,0 +1,71 @@
+const assert = require('assert');
+const {WebElement} = require('selenium-webdriver');
+const MockServer  = require('../../../../lib/mockserver.js');
+const CommandGlobals = require('../../../../lib/globals/commands-w3c.js');
+const common = require('../../../../common.js');
+const Element = common.require('element/index.js');
+
+describe('element().isPresent() command', function() {
```

Comment:
These tests are passing even if we set the `suppressNotFoundErrors` property to `false` in the `isPresent.js` file. We must also include at least one test here which fails in such a case to make sure that our unit tests are testing what they are supposed to test, that we do not throw an error in case of `isPresent()` command.

Reply:
added the test.